.env.example

```diff
@@ -119,6 +119,12 @@ NEXT_PUBLIC_SENDGRID_SENDER_NAME=
 # Used for capturing exceptions and logging messages
 NEXT_PUBLIC_SENTRY_DSN=
 
+# Formbricks Experience Management Integration
+# [apiHost]+[environmentId]
+FORMBRICKS=
```

Comment:
does this have to be FORMBRICKS or NEXT_PUBLIC_FORMBRICKS ?

also, doesn't this need to be added to turbo.json?